cron-jobs/checkForLastLogin.js

```diff
@@ -31,4 +31,5 @@ const checkLastLogin = async () => {
   )
 }
 
-module.exports = checkUsersForLastLogin
+exports.checkUsersForLastLogin = checkUsersForLastLogin
```

Comment:
why did you change export? change it back. everything must follow the same schema

Reply:
i have changed exports to module.exports

Reply:
can i resolve conersation?